be1-go/README.md

```diff
@@ -12,13 +12,15 @@ NAME:
 
 USAGE:
    pop organizer command [command options] [arguments...]
+   pop witness command [command options] [arguments...]
 
 COMMANDS:
-   serve    start the organizer server
+   serve    start the organizer or witness server
    help, h  Shows a list of commands or help for one command
 
 OPTIONS:
    --public-key value, --pk value  base64 encoded organizer's public key
+   --org-address, --org            organizer's IP address for witness to connect to organizer
```

Comment:
Yes I need to add "value", I will do that.
What do you mean by bootstrap addresses of other witnesses?
Yes will modify the CLI to add the choice of port for witness and add the relevant documentation.

Reply:
_What do you mean by bootstrap addresses of other witnesses?_ This is a suggest for a future iteration of the code, when the witness also connect to other witnesses. To do that, the witness receives an initial set of addresses of other witnesses, which is what I meant with these bootstrap addresses. The witness connect to these initial witnesses, and then through communication potentially learns of new witnesses. 
Because this is not relevant for this PR, no change is required. Just perhaps keep it in mind for the future.

Reply:
Yes understood!